### PR TITLE
Fix tint color #2182

### PR DESCRIPTION
**Pull request by onlymash**

### CODE REVIEW

File: core/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 /build
+.cxx/
```
Comment (Mygod) [inline core/.gitignore]: What's this for?

Comment (onlymash) [reply core/.gitignore]: I using AS 3.5, automatically generate this folder.

**Comment by onlymash** [comment]
Advanced is an ExpandButton, and ExpandButton doesn't provide any public methods, so we can only modify the global tint

**Comment by onlymash** [comment]
A hide class [ExpandButton](https://android.googlesource.com/platform/frameworks/support/+/c2923b8daa8f500075fad0586c9901d44b49ded5/preference/src/main/java/androidx/preference/ExpandButton.java)

**Comment by Mygod** [comment]
Can we try replacing ic_arrow_down_24dp?

File: core/.gitignore (hunk 1 above)
Comment (Mygod) [reply core/.gitignore]: What are the steps to reproduce this issue?

Comment (onlymash) [reply core/.gitignore]: AS 3.5 NDK 20.0.5392854-beta2
[.cxx dir](https://i.loli.net/2019/04/25/5cc1d2e734918.png)

Comment (Mygod) [reply core/.gitignore]: Okay. I'll check again when NDK r20 goes stable.

File: plugin/build.gradle [hunk 2]
```diff
@@ -52,6 +52,7 @@ dependencies {
     api 'androidx.core:core-ktx:1.0.1'
     api 'com.google.android.material:material:1.1.0-alpha05'
     api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+    api "com.takisoft.preferencex:preferencex-simplemenu:$preferencexVersion"
```
Comment (Mygod) [inline plugin/build.gradle]: Why?

Comment (onlymash) [reply plugin/build.gradle]: PreferenceThemeOverlay depend preference

Comment (Mygod) [reply plugin/build.gradle]: In that case, moving the fix to core is preferred.

**Pull request closed without merging**